app/src/services/operators.ts

```diff
@@ -0,0 +1,146 @@
+import BN from 'bignumber.js'
+import { AbiItem } from 'web3-utils'
+import { gql } from '@graphql-mesh/utils'
+import { operatorFactoryABI, tokenABI, operatorABI } from '@streamr/network-contracts'
+import { getBuiltGraphSDK } from '$app/../.graphclient'
+import { getConfigForChainByName, getConfigForChain } from '$shared/web3/config'
+import getDefaultWeb3Account from '$utils/web3/getDefaultWeb3Account'
+import networkPreflight from '$utils/networkPreflight'
+import { getWalletWeb3Provider } from '$shared/stores/wallet'
+import { Address } from '$shared/types/web3-types'
+
+const getOperatorChainId = () => {
+    // TODO: add to .toml
+    const sponsorshipChainName = 'dev1'
+    const chainConfig = getConfigForChainByName(sponsorshipChainName)
+    return chainConfig.id
+}
+
+gql`
```

Comment:
why this is not being assigned to any const/var? what does it actually do then?

Reply:
`npm run build-graph-client` command goes through these `gql` templates and generates typed functions for said queries. Then it's available as a function through `getBuiltGraphSDK()`.

Reply:
It's `npm run codegen` now and it generates a single file. The mechanics stays the same.